### ISSUE TITLE
how is the count for rows of test data calculated in the serenity report?

### ISSUE THREAD

**Issue by sudheer1437** [serenity-bdd/serenity-core#2765]
In the Serenity BDD report, under the overall test results tab, I am able to validate the test cases count but unable to confirm the including xx rows to test data count.

How is this count calculated?
![image](https://user-images.githubusercontent.com/4214633/161198725-3ecc4e01-2023-4b50-ad67-f42a630f5b39.png)

**Comment by wakaleo** [comment]
The rows of test data include rows in scenario outlines or parameterised JUnit tests.

**Comment by onlybiju** [comment]
Hi @wakaleo ,
Is there any way in Summary.txt report if can get the "Number of scenarios"? Currently it doesn't show anywhere in Summary report

Serenity report generated 01-04-2022 03:30:51

Test Cases:         41  (including 82 rows of test data)
Passed:             37
Failed:             2
Failed with errors: 2
Compromised:        0
Pending:            0
Ignored:            0
Skipped:            0

**Comment by sudheer1437** [comment]
Got it @wakaleo thank you

**Comment by wakaleo** [comment]
> Hi @wakaleo ,
> 
> Is there any way in Summary.txt report if can get the "Number of scenarios"? Currently it doesn't show anywhere in Summary report
> 
> 
> 
> Serenity report generated 01-04-2022 03:30:51
> 
> 
> 
> Test Cases:         41  (including 82 rows of test data)
> 
> Passed:             37
> 
> Failed:             2
> 
> Failed with errors: 2
> 
> Compromised:        0
> 
> Pending:            0
> 
> Ignored:            0
> 
> Skipped:            0

Yes, the easiest way is to update the code to include this info and propose a pull request - the API is fairly self-explanatory.
